src/util.cc

```diff
@@ -76,4 +77,10 @@ BufferValue::BufferValue(Isolate* isolate, Local<Value> value) {
   }
 }
 
+void LowMemoryNotification() {
+  if (v8_initialized) {
+    v8::Isolate::GetCurrent()->LowMemoryNotification();
```

Comment:
Can you check that `v8::Isolate::GetCurrent() != nullptr` in case it's called from a different thread?